app/views/documents/_index.html.erb

```diff
@@ -21,15 +21,16 @@
               <table id="documents" class="table table-sm table-hover">
                 <thead>
                   <tr>
-                    <th scope="col">Nom du document</th>
+                    <th scope="col"><%= link_to "Name", documents_path(sort: "filename", direction: "DESC", category: category.to_s), class:"name desc", id:"sort-name", remote: true %></th>
                     <th scope="col">Télécharger</th>
+                    <th scope="col"><%= link_to "Date", documents_path(sort: "created_at", direction: "ASC", category: category.to_s), class:"created_at", id:"sort-date", remote: true %></th>
                   </tr>
                 </thead>
 
                 <tbody>
                   <%= render partial: "documents/document", collection: @documents.select {|document|
                     document.category == category 
-                  } %>
+                  }, id: "documents" %>
```

Comment:
for method calls with multiline arguments, please align them together, and write one argument per line. This makes reading and reviewing easier, as we can see at a glance what are the arguments, and what is actually changed in a git diff

e.g.:
```ruby
<%= render partial: "documents/document",
           collection: @documents.select { |document| document.category == category },
           id: "documents" %>
```
-> we can see without reading that there are 3 arguments, and futures changes on this will be more obvious to review